JeffShepherd/Rancid-Tomatillos
# Feature/data cleaning

## What does this PR do?

- Add a utilities file that includes function to scrub data of unused properties.

## How should it be tested in the UI?

- cd into Rancid-Tomatillos repo, run npm start
- Conduct full regression of application. Ensure all features/values display correctly in the home and movie details views.
- Run all cypress tests and ensure all complete successfully.
